core/templates/components/angular-html-bind/angular-html-bind-wrapper.directive.ts

```diff
@@ -38,6 +38,8 @@ angular.module('oppia').directive('angularHtmlBindWrapper', [
         '$rootScope', '$scope',
```

Comment:
Hi @anika-001, please check the Files Changed tab to make sure that your changes are what you expect them to be. It looks like there are some files that shouldn't be there; please fix the PR accordingly before requesting a review. Thanks!

Reply:
Hi, @seanlip , I've pulled the changes and my branch is now up to date with oppia:develop.